### PR TITLE
Update go.mod and go mod tidy

### PR DESCRIPTION
**Pull request by ZenGround0**
@Stebalien renamed module to match path as prereq for fixing import paths.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,13 +1,12 @@
-module github.com/ipfs/go-hamt-ipld
+module github.com/filecoin-project/go-hamt-ipld
 
 require (
 	github.com/ipfs/go-block-format v0.0.2
 	github.com/ipfs/go-cid v0.0.6
+	github.com/ipfs/go-hamt-ipld v0.1.1
 	github.com/ipfs/go-ipld-cbor v0.0.4
-	github.com/ipfs/go-ipld-format v0.0.2 // indirect
```
Comment (ZenGround0) [inline go.mod:7]: `go mod tidy` removed these, should I keep them around?

Comment (Stebalien) [reply go.mod:7]: That's fine.

File: go.mod [hunk 2]
```diff
@@ -1,13 +1,12 @@
-module github.com/ipfs/go-hamt-ipld
+module github.com/filecoin-project/go-hamt-ipld
 
 require (
 	github.com/ipfs/go-block-format v0.0.2
 	github.com/ipfs/go-cid v0.0.6
+	github.com/ipfs/go-hamt-ipld v0.1.1
```
Comment (Stebalien) [inline go.mod:6]: This is wrong. Grep the repo for imports that need to be fixed.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ go-hamt-ipld
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
 [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://libp2p.io/)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
```
Comment (Stebalien) [inline README.md:6]: Let's fix these at the same time.

**Pull request merged**